filebeat/module/elasticsearch/gc/test/gc.log

```diff
@@ -47,7 +47,7 @@
 [2018-06-13T07:44:24.343+0000][32376][gc,heap      ] GC(0) ParNew: 279616K->17562K(314560K)
 [2018-06-13T07:44:24.343+0000][32376][gc,heap      ] GC(0) CMS: 0K->0K(699072K)
 [2018-06-13T07:44:24.343+0000][32376][gc,metaspace ] GC(0) Metaspace: 22819K->22819K(1071104K)
-[2018-06-13T07:44:24.343+0000][32376][gc           ] GC(0) Pause Young (Allocation FailurGe) 273M->17M(989M) 13,344ms
```

Comment:
This is a log file from Elasticsearch and should not be modified.

Reply:
Will drop